From f46717606ad923b3c9748d18b9555f48443a5829 Mon Sep 17 00:00:00 2001
From: Jason Wessel <jason.wessel@windriver.com>
Date: Wed, 3 Feb 2010 13:51:19 -0500
Subject: [PATCH 2/7] powerpc: special load offset for qemu

Patch the load offset to work correctly with qemu_boot_loader.

Signed-off-by: Jason Wessel <jason.wessel@windriver.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 arch/powerpc/kernel/vmlinux.lds.S |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/powerpc/kernel/vmlinux.lds.S b/arch/powerpc/kernel/vmlinux.lds.S
index f096e72..f3a15c9 100644
--- a/arch/powerpc/kernel/vmlinux.lds.S
+++ b/arch/powerpc/kernel/vmlinux.lds.S
@@ -8,6 +8,11 @@
 #include <asm/cache.h>
 #include <asm/thread_info.h>
 
+#ifdef CONFIG_PPC_QEMU
+#undef LOAD_OFFSET
+#define LOAD_OFFSET 0
+#endif /* CONFIG_PPC_QEMU */
+
 ENTRY(_stext)
 
 PHDRS {
-- 
1.7.9.5

